index.html

```diff
@@ -34,6 +35,185 @@ <h1 class="text_title item-1">Vanessa Bonito</h1>
+                    </div>
+                </div>
+                <div class="flex-button-work">
+                    <div class="button-work">
+                        See this project
+                    </div>
+                    <div class="arrow_icon flex-button-icon"></div>
+                </div>
+            </div>
+            <div class="container-work" id="project5">
+                <div class="img-work img-work-source5"></div>
+                <div class="div-title flex-work-title">
+                    <h1 class="text_titleH3">Project 5</h1>
+                </div>
+                <div class="flex-ul-work">
+                    <div class="ul-work-item">
+                        <ul class="ul-style">
+                            <li class="li-style">HTML/CSS</li>
+                            <li class="li-style">Ruby on rails</li>
+                            <li class="li-style">JavaScript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="flex-button-work">
+                    <div class="button-work">
+                        See this project
+                    </div>
+                    <div class="arrow_icon flex-button-icon"></div>
+                </div>
+            </div>
+            <div class="container-work" id="project6">
+                <div class="img-work img-work-source6"></div>
+                <div class="div-title flex-work-title">
+                    <h1 class="text_titleH3">Project 6</h1>
+                </div>
+                <div class="flex-ul-work">
+                    <div class="ul-work-item">
+                        <ul class="ul-style">
+                            <li class="li-style">HTML/CSS</li>
+                            <li class="li-style">Ruby on rails</li>
+                            <li class="li-style">JavaScript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="flex-button-work">
+                    <div class="button-work">
+                        See this project
+                    </div>
+                    <div class="arrow_icon flex-button-icon"></div>
+                </div>
+            </div>
+        </section>
+        <section class="grid-about">
+            <div class="div-title">
+                <h1 class="text_titleH2">About</h1>
+            </div>
+            <div class="card-about">
+                <p class="text_style">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+                    et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                    aliquip ea eiusmod
+                </p>
+                <div class="flex-button-about dark_button">
+                    <div class=".button-about">
+                        Get my CV
+                    </div>
```

Comment:
Let's make this a `button `or a link tag, so when the user clicks, he gets your CV.